eqcorrscan/utils/plotting.py

```diff
@@ -2207,6 +2207,135 @@ def subspace_fc_plot(detector, stachans, **kwargs):
     return fig
 
 
+@additional_docstring(plotting_kwargs=plotting_kwargs)
+def twoD_seismplot(catalog=None, locations=None, bgcolor='#909090',
+                   method='depth', **kwargs):
+    """
+    Plot seismicity in a 2D map with two cross section along latitude and
+    longitude.
+
+    :type catalog: obspy.core.event.catalog.Catalog
+    :param catalog: Obspy catalog class containing event metadata
+    :type locations: list
+    :param locations:
+        list of one tuple per event of (lat, long, depth, time) with
+        down positive.
+    :type bgcolor: string
+    :param bgcolor: Background's color of map and sections.
+        all name or RGB code that acceptable in matplotlib.
+    :type method: string
+    :param method:
+        making color palette of locations according to 'depth', 'time' or
+        'sequence'.
+    {plotting_kwargs}
+
+    :returns: :class:`matplotlib.figure.Figure`
+
+    .. note::
+        If each location doesn't have time or depth, set them to zero.
+    .. note::
+        kwargs accepts all option that available in
+        `matplotlib.axes.Axes.scatter`.
+    """
+    import matplotlib.pyplot as plt
+    from matplotlib import gridspec
+    from mpl_toolkits.axes_grid1 import make_axes_locatable
+    assert (catalog and locations) or catalog or locations,\
+        "Requires catalog and/or locations"
+    # set default parameters of plt.scatter()
+    default_parameters = {'cmap': 'jet_r', 'marker': ',', 's': 1, 'lw': 1}
```

Comment:
```suggestion
    default_parameters = {'cmap': 'viridis_r', 'marker': ',', 's': 1, 'lw': 1}
```
Jet and rainbow are not very friendly color palettes - people seem to like viridis.